Java/core/src/main/java/com/amazon/randomcutforest/RandomCutForest.java

```diff
@@ -568,11 +569,16 @@ public float[] lastShingledPoint() {
 
     /**
      *
-     * @return the sequence index of the last known shingled point
+     * @return the sequence index of the last known shingled point if internal
```

Comment:
If shingle-size is 10 and internal shingling is disabled, then `nextSequenceIndex` should always return 9?

Reply:
Fixed. I was confused/forgot what the function was doing.